odoo/tools/image.py

```diff
@@ -3,18 +3,13 @@
 import base64
 import binascii
 import io
-from typing import Tuple, Union
-
-from PIL import Image, ImageOps
-# We can preload Ico too because it is considered safe
-from PIL import IcoImagePlugin
-try:
-    from PIL.Image import Transpose, Palette, Resampling
-except ImportError:
-    Transpose = Palette = Resampling = Image
 
+from typing import Union
 from random import randrange
 
+from PIL import Image, ImageOps, IcoImagePlugin  # noqa: F401
```

Comment:
You must ensure here that patches are called before you import these lines because monkeypatching is not forced before importing tools.